electron/package.json

```diff
@@ -68,7 +69,7 @@
         "cucumber": "^6.0.5",
         "cucumber-pretty": "^6.0.0",
         "dotenv-webpack": "^6.0.0",
-        "electron": "13",
+        "electron": "^13.1.2",
```

Comment:
any reasons to upgrade electron?

Reply:
just automatically update by some command

Reply:
I will downgrade it